packages/data-portal-commons/src/lib/getCaseValues.ts

```diff
@@ -1,5 +1,6 @@
 import _ from 'lodash';
 import { Entity } from './entity';
+import { getNormalizedOrgan } from '@htan/data-portal-commons';
```

Comment:
better to do a relative import here
```suggestion
import { getNormalizedOrgan } from './entityReportHelpers';
```

Reply:
Done.